firefox-ios/Client/Frontend/Settings/SettingsTableViewController.swift

```diff
@@ -746,6 +725,32 @@ class CheckmarkSetting: Setting {
         }
     }
 
+    private func setupLeftCheckLabel(_ cell: UITableViewCell, theme: Theme) {
+        check.font = FXFontStyles.Regular.body.scaledFont()
+        let checkMarkWidth = check.font.pointSize + UX.checkmarkLeading
+        cell.contentView.directionalLayoutMargins = NSDirectionalEdgeInsets(top: 0.0,
+                                                                            leading: checkMarkWidth,
+                                                                            bottom: 0.0,
+                                                                            trailing: 0.0)
+        let checkColor = isChecked() ? theme.colors.actionPrimary : UIColor.clear
+        let checkmarkSize = UIFontMetrics.default.scaledValue(for: UX.checkmarkHeight)
```

Comment:
In the old code the width was different from the height. Is there a reason we are using the same value for both height and width now?

Reply:
Yes you are right, i'm going to restore the previous width and scale it